gramps/gen/lib/note.py

```diff
@@ -272,6 +273,18 @@ def get(self):
         """
         return str(self.text)
 
+    def get_preview(self):
+        """Return the max visible text string associated with the note.
+
+        :returns: either the *clear* text of the note contents or the maximum
+                  visible text.
+        :rtype: unicode
+        """
+        max_vis_len = config.get("interface.note-max-vis-length")
+        if len(str(self.text)) > max_vis_len:
```

Comment:
```suggestion
        preview = str(self.text).replace("\n", " ")
        if len(preview) > max_vis_len:
            return preview[:max_vis_len] + "..."
        return preview
```

Reply:
This may be slightly simpler and easier to read.

Reply:
Done